conda-forge.yml

```
@@ -1,3 +1,5 @@
+build_platform:
+  osx_arm64: osx_64
```

Switch to cross-compilation to avoid issues with Travis:
```suggestion
  osx_arm64: osx_64
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```